content/python/concepts/assert/assert.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'assert'
+Description: 'A keyword for confirming the truthiness of a given statement'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Assert'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+'A keyword for confirming the truthiness of a given statement that is often used to facilitate development, document code and debug.'
+
+## Syntax
+
+```pseudo
+assert some_condition, assert_message
+```
+
+The `assert` keyword is followed by the statement containing the condition to be tested, such as validating a data type or membership within a given container, etc. An optional message can be added that will be printed when the assertion error is raised.
```

Comment:
The "etc." sounds a bit abrupt here. I think it can be removed.
```suggestion
The `assert` keyword is followed by the statement containing the condition to be tested, such as validating a data type or membership within a given container. An optional message can be added that will be printed when the assertion error is raised.
```